static/styles/right_sidebar.css

```diff
@@ -125,6 +130,26 @@
     cursor: pointer;
 }
 
+#realm-user-default-settings {
+    .user-name-and-status-emoji {
+        display: block;
+        width: 100%;
+        height: 20px;
+
+        .user-name {
+            display: inline-block;
+            width: calc(100% - 24px);
```

Comment:
Can we make this 24px a constant that's defined somewhere?

Reply:
Done!